inc/class-book.php

```diff
@@ -191,6 +192,82 @@ static function getBookInformation( $id = null ) {
 		return $book_information;
 	}
 
+	/**
```

Comment:
Honestly, those new methods deserve a new class. But since it is temporal I decided to keep it in the main book class.